com.dynamo.cr/com.dynamo.cr.common/build.gradle

```diff
@@ -0,0 +1,76 @@
+import java.util.concurrent.Executors
+import java.util.concurrent.Callable
+
+apply plugin: 'java'
+apply from: '../com.dynamo.cr.bob/basic.gradle'
+
+task cleanGenerated {
+    doLast {
+        delete 'generated'
+    }
+}
+
+clean.dependsOn cleanGenerated
+
+task extGeneratedCopy(dependsOn: [getGitRevision]) {
+    doLast {
+        def gitRevision = tasks.getGitRevision.gitRevision
+        def extGeneratedDir = file('ext_generated')
+        extGeneratedDir.mkdirs()
+        copy {
+            from "${project.dynamoHome}/archive/${gitRevision}/engine/share/java"
+            include 'dlib.jar'
+            into extGeneratedDir
+        }
+        copy {
+            from "${project.dynamoHome}/share/java"
+            include 'dlib.jar'
+            into extGeneratedDir
+        }
+        if (extGeneratedDir.listFiles().length == 0) {
+            throw new RuntimeException("dlib.jar is missing")
+        }
+    }
+}
+
+def protoTasks = [
+        [dir: "proto", file: "cr_ddf.proto"],
+        [dir: "proto", file: "cr_protocol_ddf.proto"],
```

Comment:
These two used by tests, and we need them to be able to run a single test. But the rest is just a duplicate of the main bob folder proto files. Having these classes in both folders end up in "duplicate symbols" error. So I removed them, and keep only unique proto files